### PR TITLE
fix(analytical): fix louvain change unit test min_progress=0, progress_tries=2

### PR DESCRIPTION
**Pull request by JackyYangPassion**
## What do these changes do?

change louvain  unit test min_progress=0, progress_tries=2

## Related issue number

Fixes

### CODE REVIEW

**Review by sighingnow — APPROVED** [review]

File: analytical_engine/apps/pregel/louvain/louvain_app_base.h [hunk 1]
```diff
@@ -273,13 +273,6 @@ class LouvainAppBase
       } else if (ctx.compute_context().superstep() == compress_community_step) {
         ctx.GetVertexState(v).is_alived_community = false;
       }
-
-      if (!ctx.compute_context().active(v)) {
-        std::vector<std::pair<vid_t, edata_t>> tmp_edges;
-        ctx.GetVertexState(v).fake_edges.swap(tmp_edges);
-        std::vector<vid_t> tmp_nodes;
-        ctx.GetVertexState(v).nodes_in_community.swap(tmp_nodes);
-      }
```
Comment (siyuan0322) [inline analytical_engine/apps/pregel/louvain/louvain_app_base.h:282]: Why delete this? this is for reduce the memory usage by dropping unused fields.

File: python/graphscope/tests/unittest/test_app.py [hunk 2]
```diff
@@ -234,8 +234,10 @@ def test_other_app_on_undirected_graph(
     assert np.allclose(r, triangles_result["undirected"])
 
     # louvain
-    ctx = louvain(p2p_project_undirected_graph, min_progress=50, progress_tries=2)
-    assert ctx is not None
+    ctx = louvain(p2p_project_undirected_graph, min_progress=0, progress_tries=2)
+    df = ctx.to_dataframe({"node": "v.id", "r": "r"})
+    community_num = len(df["r"].unique())
+    assert community_num == 103
```
Comment (siyuan0322) [inline python/graphscope/tests/unittest/test_app.py:240]: Is this number consistent over multiple runs?

Comment (acezen) [reply python/graphscope/tests/unittest/test_app.py:240]: it should be consistent and still trying to figure out what make the result mismatch

**Comment by JackyYangPassion** [comment]
#3472  has update so close this pr

**Pull request closed without merging**